website/docs/r/cognitive_account.html.markdown

```diff
@@ -138,6 +138,8 @@ A `storage` block supports the following:
 
 * `identity_client_id` - (Optional) The client ID of the managed identity associated with the storage resource.
 
+~> **NOTE:** Not all `kind` support a `storage` block. For example the `kind` `OpenAI` does not support it.
```

Comment:
@zioproto is there some MSFT documentation that we can link here with a definitive list of what cognitive account kinds support storage? If not then I think we should rephrase this to be more specific
```suggestion
~> **NOTE:** Cognitive Accounts with `kind` `OpenAI` do not support a `storage` block.
```

Reply:
@felixwa Are you able to help me with this ask ? Is it just `OpenAI` that does not support this ? Thanks

Reply:
@stephybun it seems there is not a MSFT documentation page to link that publishes this definitive list. Should we just keep the generic sentence ?
```
Not all `kind` support a `storage` block.
```

I added the OpenAI example because it is very popular and a lot of customers ask about what the storage block is for.

Please advice what changes are necessary to get this merged. Thanks